modules/MotionParser.cpp

```diff
@@ -351,6 +351,27 @@ vector<Motion*> MotionParser::createMotions(Robot& robot, string& commandFilePat
         break;
       }
 
+      // PCIDS: 画像ラインを用いた距離停止直進
```

Comment:
画像ラインってのが少し分かりづらいかな。Notion見てみたら、カメラ映像(画像)の上部に赤色のバウンディングボックスがあって、その中に青丸が入り込んだら停止するって感じかな？
僕の認識が合ってれば、コメントこんな感じにした方がいいかも。
```suggestion
      // PCIDS: IMU角度補正直進に、カメラ画像の色検出による停止条件を追加した動作
```

Reply:
コメントの部分は修正しておきました。